Makefile

```diff
@@ -117,6 +117,35 @@ coverage: ## generate coverprofiles from the unit tests
 		fi; \
 	done )
 
+.PHONY: test-cloud-storage
+test-cloud-storage: start-cloud-storage run-s3-tests stop-cloud-storage ## run cloud storage driver tests
+
+.PHONY: start-cloud-storage
+start-cloud-storage: ## start local cloud storage (minio)
+	docker compose -f tests/docker-compose-storage.yml up minio minio-init -d
```

Comment:
We should make it possible to override the compose tool.

Something like this at the top of the Makefile:

```
# Tools
COMPOSE=docker compose
```

Then:
```suggestion
	$(COMPOSE) -f tests/docker-compose-storage.yml up minio minio-init -d
```